charforge-gui/backend/app/api/settings.py

```
@@ -0,0 +1,233 @@
+    
+    return EnvironmentSettingsResponse(
+        HF_TOKEN=settings.get('HF_TOKEN', ''),
+        HF_HOME=settings.get('HF_HOME', ''),
+        CIVITAI_API_KEY=settings.get('CIVITAI_API_KEY', ''),
+        GOOGLE_API_KEY=settings.get('GOOGLE_API_KEY', ''),
+        FAL_KEY=settings.get('FAL_KEY', '')
+    )
+
+@router.post("/test-environment")
+async def test_environment_settings(
+    db: Session = Depends(get_db),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)